Topics/lifecycle.md

```diff
@@ -0,0 +1,111 @@
+# The Data Science Project lifecycle
+
+# What you will learn 
+In this guide you will understand the big picture of the Data Science project lifecycle, why and how to perform each step, from getting the data to putting predictive models in production. You can then use this guide as a "checklist" of to-do steps to perform during your projects. 
+
+## Prerequisites
+There are no prerequisites for this guide.
+
+## Time to complete
+30 minutes.
+
+# Index
+- [How to use this guide](#How-to-use-this-guide)
+- [Looking at the big picture](#Looking-at-the-big-picture)
+  - [Frame the problem](#Frame-the-problem)
+  - [Tools that can help in the definition of the problem](#Tools-that-can-help-in-the-definition-of-the-problem)
+  - [Select a performance measure](#Select-a-performance-measure)
+  - [Check the assumptions](#Check-the-assumptions)
+- [Get the data](#Get-the-data)
+  - [Create the workspace](#Create-the-workspace)
+  - [Download the data](#Download-the-data)
+  - [Discover data and get domain experts insights](#Discover-data-and-get-domain-experts-insights)
+  - [Create a test set](#Create-a-test-set)
+- [Prepare the data for machine mearning algorithms](#Prepare-the-data-for-machine-mearning-algorithms)
```

Comment:
Feature Selection and Feature Engineering is missing,  have a look at our Trello card, and see if we have all the high-level categories mentioned on there.